### PR TITLE
Make things more explicit

### PR DESCRIPTION
**Pull request by sultaniman**
Unwrapped comparison into more explicit conditions

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,8 +1,17 @@
 'use strict';
+
```
Comment (arthurvr) [inline index.js]: Can you please not change unrelated things? This was intentionally not there. :wink:

File: index.js [hunk 2]
```diff
@@ -1,8 +1,17 @@
...
 module.exports = function (x, min, max) {
 	if (min > max) {
 		throw new RangeError('`min` should be lower than `max`');
 	}
 
-	return x < min ? min : x > max ? max : x;
+	if (x < min) {
+		x = min;
```
Comment (arthurvr) [inline index.js]: Just `return min;` directly instead of changing `x`.

File: index.js [hunk 3]
```diff
@@ -1,8 +1,17 @@
...
+	}
+
+	if (x > max) {
+		x = max;
```
Comment (arthurvr) [inline index.js]: Just `return max;` directly instead of changing `x`.

File: index.js (hunk 1 above)
Comment (sultaniman) [inline index.js]: Sure :smile:

**Comment by arthurvr** [comment]
Yeah, I personally like it more if it's written out like this. WDYT @sindresorhus?

**Comment by sindresorhus** [comment]
Sure

**Pull request merged**